docker/crons.conf

```diff
@@ -0,0 +1,2 @@
+* * * * * /bin/bash /usr/bin/restart.sh
+* * * * * echo "Hello world" >> /usr/src/app/phantomjs.log
```

Comment:
Thanks, but that link says nothing about `/etc/crons.conf`. Cron looks at `/etc/crontab`, which in turn often includes `/etc/cron.d/*`, `/etc/cron.daily/*`, etc.